frontend/packages/dev-console/src/components/health-checks/HealthChecksPage.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { FirehoseResource, Firehose } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ServiceModel as KnativeServiceModel } from '@console/knative-plugin';
+import AddHealthChecksForm from './AddHealthChecksForm';
+
+type HealthChecksProps = RouteComponentProps<{
+  ns: string;
+  kind: string;
+  name: string;
+  containerName: string;
+}>;
+
+const HealthChecksPage: React.FC<HealthChecksProps> = ({ match }) => {
+  const { ns, kind, name, containerName } = match.params;
+  const resource: FirehoseResource[] = [
+    {
+      kind: kind === KnativeServiceModel.kind ? referenceForModel(KnativeServiceModel) : kind,
```

Comment:
If your URL was properly qualified you'd have the correct kind. Right now your kind is just `Service` which is the wrong Service. 
```suggestion
      kind: referenceForModel(kind),
```

You can tell because the link to the workload for a knative service is this:
![image](https://user-images.githubusercontent.com/14068621/79678722-53f38080-81cc-11ea-8f0d-3124f7e41887.png)

Which actually brings you to the wrong page.

Reply:
Fixed it. 
<img width="607" alt="Screenshot 2020-04-19 at 12 09 02 PM" src="https://user-images.githubusercontent.com/2561818/79681372-7bcbfe00-8237-11ea-8bbd-79c7e2e950ef.png">